bindep.txt

```diff
@@ -1,6 +1,10 @@
-libxml2-devel [platform:rpm]
+openssl-devel [platform:rpm compile]
+gcc [platform:rpm compile]
+libcurl-devel [platform:rpm compile]
+libxml2-devel [platform:rpm compile]
 python3-pycurl [platform:rpm]
 python3-netaddr [platform:rpm]
 python3-jmespath [platform:rpm]
-python3-passlib [platform:rpm]
+python3-passlib [platform:fedora]
+python3-passlib [platform:centos epel]
 qemu-img [platform:rpm]
```

Comment:
Could you also please add python3-ovirt-engine-sdk4? This is the most important dependency, without it no ovirt module coult work. But we need oVirt 4.4 repositories (or RHV 4.4 Tools channel) to make it available:

https://plain.resources.ovirt.org/pub/ovirt-4.4/rpm/el8/

Reply:
This is being specified via this collection's [requirements.txt](https://github.com/oVirt/ovirt-ansible-collection/blob/master/requirements.txt)

Reply:
The requirements.txt is used to install the package from PyPI, right?
I think it would be better to install from RPM.

Reply:
yes, requirements.txt is for packages pulled from pypi

Reply:
The requirements.txt file already exists in this repo. Changing this seems out of scope for what this PR is trying to accomplish. 

For our usage, we will be using Cachito downstream to install from pip, which obviates the need for RPMs.

Reply:
@mnecas  To further elaborate on my position here... this file did not exist until https://github.com/oVirt/ovirt-ansible-collection/pull/91, which was opened by my colleague @AlanCoding. You approved that PR. 

This metadata was added for the purpose of being read by Ansible Builder in order to produce an Execution Environment. We do not currently have a good mechanism in place to rely on dependencies that are not in the base CentOS or EPEL repos. I would very much prefer to _not_ go down that path, because it further distances us from our ultimate goal of having a Dockerfile that we can build both upstream and downstream.

Reply:
Yeah, I understand you and know about the #91. If there is no possibility to use our repo and we can use only EPEL, I don't mind using the PyPI, not best but what else can we do.
I'll need to dig more in the Ansible Builder, don't know much about it now.

Reply:
Longer term, maybe we should open an issue in ansible-builder for allowing collections to specify custom repos? Shorter term, it would still help us to get this in so that we can go back to testing with the released version of this collection, which is required for Ansible 2.9 that doesn't support collection SCM installs.

Reply:
I checked the issues and [1] looks similar, so we don't need to create a new issue.
I'm going to merge this as the short-term solution and after the proper changes on the builder, we can update the collection once again.
[1] https://github.com/ansible/ansible-builder/issues/159